docker/runner.Dockerfile

```diff
@@ -60,6 +60,31 @@ RUN mkdir -p /home/runner/.ssh \
 # Adjust permissions
 RUN chown -R runner:runner /home/runner
 
+# Setup podman
+ENV APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE="1"
+RUN echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_20.04/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list && \
+    curl -o key https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_20.04/Release.key && \
+    apt-key add key && \
+    apt-get update -qq && \
+    apt-get install -y software-properties-common uidmap fuse-overlayfs podman
+
+RUN useradd podman; \
+    echo podman:10000:5000 > /etc/subuid; \
+    echo podman:10000:5000 > /etc/subgid;
+
+VOLUME /var/lib/containers
+VOLUME /home/podman/.local/share/containers
+
+ADD https://raw.githubusercontent.com/containers/image_build/main/podman/containers.conf /etc/containers/containers.conf
+ADD https://raw.githubusercontent.com/containers/image_build/main/podman/podman-containers.conf /home/podman/.config/containers/containers.conf
+
+RUN chown podman:podman -R /home/podman
+
+RUN chmod 644 /etc/containers/containers.conf; sed -i -e 's|^#mount_program|mount_program|g' -e '/additionalimage.*/a "/var/lib/shared",' -e 's|^mountopt[[:space:]]*=.*$|mountopt = "nodev,fsync=0"|g' /etc/containers/storage.conf
```

Comment:
might be better to just create the entire config and commit it to our repo instead of doing complex replacements

Reply:
Added a link to blogpost